examples/alias_migration.py

```diff
@@ -38,24 +38,28 @@
 import os
 from datetime import datetime
 from fnmatch import fnmatch
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
 
-from elasticsearch_dsl import Date, Document, Keyword, Text, connections
+from elasticsearch_dsl import Document, Keyword, M, connections, mapped_field
 
 ALIAS = "test-blog"
 PATTERN = ALIAS + "-*"
 
 
 class BlogPost(Document):
-    title = Text()
-    published = Date()
-    tags = Keyword(multi=True)
-    content = Text()
+    if TYPE_CHECKING:
+        _id: int
```

Comment:
This is a trick to get the dataclass support to accept these extra arguments in the constructor.

Reply:
Since this is an example, should we explain the trick?

Reply:
Yes, I should add a comment here, thanks.